avahi-common/alternative.c

```diff
@@ -109,6 +113,13 @@ char *avahi_alternative_host_name(const char *s) {
         avahi_free(c);
     }
 
+    alt = alternative;
+    len = sizeof(alternative);
+    ret = avahi_escape_label(r, strlen(r), &alt, &len);
```

Comment:
`ret` should probably be checked too.

Reply:
It can fail only if there isn't enough room and it can always fit into `alt` in its current form. The fuzz target should trigger the assertion if there is anything wrong here.